src/main/services/auth.ts

```diff
@@ -54,10 +57,8 @@ export class AutomationCloudAuthService extends AuthService {
         const authorization = ctx.req.headers[authHeaderName] || '';
         // check auth header(jwt) supplied by gateway
         if (authorization) {
-            this.logger.info('[auth-service] internal auth header detected');
```

Comment:
Nice 👍 I saw this in tests, so was wondering if the logging is a bit on a verbose side —glad we made it quieter.

Reply:
Yes, it was a temporary verbose logging for the testing we did today  🎉